openpype/modules/kitsu/kitsu_module.py

```diff
@@ -128,15 +128,35 @@ def push_to_zou(login, password):
 @click.option(
     "-p", "--password", envvar="KITSU_PWD", help="Password for kitsu username"
 )
-def sync_service(login, password):
+@click.option(
+    "-prj",
```

Comment:
Shouldn't these be:
```py
"-prjs",
"--projects"
```

Reply:
The arg `--project` accept one input parameter not many parameters

Reply:
Ah I see, the last "projects" in line `134` would collect any multiple `-prj` provided?